test/command/suite/select/filter/near/min_interval/with_max/match.expected

```diff
@@ -0,0 +1,47 @@
+table_create Memos TABLE_PAT_KEY ShortText
+[[0,0.0,0.0],true]
+column_create Memos content COLUMN_SCALAR ShortText
+[[0,0.0,0.0],true]
+table_create Terms TABLE_PAT_KEY ShortText   --default_tokenizer TokenBigram   --normalizer NormalizerAuto
+[[0,0.0,0.0],true]
+column_create Terms memos_content COLUMN_INDEX|WITH_POSITION Memos content
+[[0,0.0,0.0],true]
+load --table Memos
+[
+{"_key":"alphabets", "content": "a b c d e f g i"}
+]
+[[0,0.0,0.0],1]
+select --table Memos --match_columns content --query '*N3,,3 "a e"'
```

Comment:
Could you use different values for `max_interval` and `min_interval`?

Reply:
Fixed.